src/platform/unittest/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ set(SOURCES
     main.cpp
     CryptTest.cpp
     DataPathTest.cpp
+    Platform.cpp
```

Comment:
Please call this PlatformTest.cpp, not Platform.cpp please.

Reply:
make sense. updated.